cardinal/gamestate/doc.go

```diff
@@ -65,7 +65,7 @@ value:  JSON serialized bytes that can be deserialized to a list of transactions
 processed in the last started tick. This data is only relevant when the START-TICK number does not match the END-TICK
 number.
 
-# In-memory storage model
+# In-memory dbStorage model
```

Comment:
The changes from `storage` to `dbStorage` in this file should probably go back to `storage`. Looks like this comes from the refactor tool